src/components/02-elements/cookie-consent/cookie-consent.scss

```diff
@@ -1,7 +1,9 @@
 .cookie-consent {
```

Comment:
Did you mean to commit these changes in this PR? I thought it was done in #62

Reply:
This branch was checkout from the #62 but was done before the `index` was added. That is why there is a conflict here. Let me fix this first.

Reply:
@michaelwood I have fixed the conflict so the file is no longer in this PR.